README.md

```diff
@@ -1 +1,53 @@
-# Metrics-Webapp
\ No newline at end of file
+# Metrics-Webapp
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Additional description about the project and its features.
+
+This project was built to fetch data from an api an map out specific companies location and their prices.
+
+![](/src/assets/navigation.PNG)
+![](/src/assets/Page1.PNG)
+![](/src/assets/Page2Style.PNG)
+![](/src/assets/test.PNG)
+
+
+# Built With
+
+1. React and Redux
+2. Linters (Stylelint, Lighthouse)
+
+# Getting Started
+
+## Installation
+1. You can run: `npm install create-react-app` (to create a new react app) 
+2. After the installation of the react app you can run: `npm start` (to load it on the server)
+3. Before you run test you first have to install `jest`in order to run test you can as well add `--coverage`to the script array in your package.json
+4. Then after the previewing in locally you can now run : `npm test` (to see if the code works perfectly.
+
+To get a local copy up and running follow these simple example steps.
+
+# Prerequisites
+
+Setup: Clone https://github.com/oluyaratosin123/Metrics-Webapp.git into the destination folder of your local machine.
+Install: Install linters with the npx commands given in the linters.yml file.
+Usage: N/A
+Run tests: Run syntax tests with the use of the npx commands given in the linters.yml file.
+Deployment: Run index.html on your local server.
+
+# Authors
+
+GitHub: https://github.com/oluyaratosin123
+LinkedIn: https://www.linkedin.com/in/edward-oluyara-708b88215/
+
+Author name: Nelcon Sakwa 
+Behance: https://www.behance.net/sakwadesignstudio
```

Comment:
- [ ] I believe you have put a lot of work into this project. In that light, it will be a good idea to deploy your project, share it will other people so that they too can make use of your app right?
Therefore, I will suggest your try to deploy your project using Heroku, Netlify or GitHub Pages and add the link to the live demo in your readme.

- [ ] I also noticed your readme or PR doesn't contain a video of you explaining your project. It is a requirement to do a video presentation of your project to give visitors some insight about your project.
You can summarize your video using the following bullet points.

1. Recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, highlighting all the project features.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.